src/chainparams.h

```diff
@@ -23,6 +23,11 @@ typedef std::map<int, uint256> MapCheckpoints;
 
 struct CCheckpointData {
     MapCheckpoints mapCheckpoints;
+
+    int GetHeight() const {
+        const auto& final_checkpoint = mapCheckpoints.rbegin();
+        return final_checkpoint->first /* height */;
```

Comment:
I think ideally, any begin/end/rbegin/rend iterator should be tested before deferencing:

```c++
return final_checkpoint == mapCheckpoints.rend() ? 0 : final_checkpoint->first;
```

or similar.